### ISSUE TITLE
Not working when API restrict to android app only!

### ISSUE THREAD

**Issue by milon87** [mziccard/android-google-cloud-translate#1]
working fine when no restriction is imposed in google console. If I set API to restrict then getting this error

Caused by: com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
                                                                                 {
                                                                                   "code" : 403,
                                                                                   "errors" : [ {
                                                                                     "domain" : "usageLimits",
                                                                                     "message" : "The request did not specify any Android package name or signing-certificate fingerprint. Please ensure that the client is sending them or use the API Console to update your key restrictions.",
                                                                                     "reason" : "ipRefererBlocked",
                                                                                     "extendedHelp" : "https://console.developers.google.com/apis/credentials?project=778234082059"
                                                                                   } ],
                                                                                   "message" : "The request did not specify any Android package name or signing-certificate fingerprint. Please ensure that the client is sending them or use the API Console to update your key restrictions."
                                                                                 }

**Comment by muhammedakiftas** [comment]
Warning:WARNING: Dependency org.apache.httpcomponents:httpclient:4.0.1 is ignored for debug as it may be conflicting with the internal version provided by Android.
Can you help me

**Comment by imosmanyilmaz** [comment]
@milon87  you are right. If  you are put a client restriction, you will be getting error

**Comment by hexicle** [comment]
@muhammedakiftas 
in `app/build.gradle`:
```
dependencies {
    compile ('com.google.cloud:google-cloud-translate:1.6.0') {
        exclude group: 'org.apache.httpcomponents', module: 'httpclient'
    }
}
```

**Comment by tobujin** [comment]
@milon87 i have that error too, so my key restriction is updated to API console but too same result.
Anybody can help me?

**Comment by tobujin** [comment]
@milon87 i have that error too, so my key restriction is updated to API console but too same result.
Anybody can help me?

**Comment by tobujin** [comment]
@milon87 i have that error too, so my key restriction is updated to API console but too same result.
Anybody can help me?

**Comment by tobujin** [comment]
@milon87 i have that error too, so my key restriction is updated to API console but too same result.
Anybody can help me?
